mmdet3d/datasets/det3d_dataset.py

```diff
@@ -246,7 +246,8 @@ def parse_data_info(self, info: dict) -> dict:
                     self.data_prefix.get('pts', ''),
                     info['lidar_points']['lidar_path'])
 
-            info['num_pts_feats'] = info['lidar_points']['num_pts_feats']
+            if 'num_pts_feats' in info['lidar_points']:
```

Comment:
Hi, I think num_pts_feats is an useful key for visualize, so can you add default num_pts_feats in convert script?
```python
temp_data_info['lidar_points']['num_pts_feats'] = 5
```
https://github.com/open-mmlab/mmdetection3d/blob/ea131ebc77bd91ecf91be8669c8702293b2390a3/tools/dataset_converters/update_infos_to_v2.py#L287